### DISCUSSION TITLE
Display a table field from another component

### DISCUSSION THREAD

**Discussion by Pantaleaogc** [joomengine/Joomla-Component-Builder#720] [Q&A] [answered]
Hi...

I want to create a field and I need that field to show an item from a table in another component.

It would be possible, I have not found where I see this possibility in the videos.

Grateful for the support.

**Comment by LordRazen** [comment, marked as answer]
As far as I understood JCB, this is possible with a "Custom Field" 
https://www.youtube.com/watch?v=VpzYbifqv0M&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&t=329s

Search for "Custom Field" in JCB -> Dashboard -> Wiki and you find several videos about this.
